#Action Goal
string device_name
---
#Action Result
bool success
diagnostic_msgs/KeyValue[] is_detected
---
#Action Feedback